packages/wallets/src/features/cashier/modules/Transfer/hooks/useTransferMessages/utils/cumulativeAccountLimitsMessageFn.ts

```diff
@@ -80,9 +80,17 @@ const cumulativeAccountLimitsMessageFn = ({
     )
         return null;
 
-    const sourceCurrencyLimit = allowedSumUSD * (activeWalletExchangeRates?.rates?.[sourceAccount.currency] ?? 1);
-
-    const sourceCurrencyRemainder = availableSumUSD * (activeWalletExchangeRates?.rates?.[sourceAccount.currency] ?? 1);
+    const USDToSourceRate =
+        activeWallet.currency === sourceAccount.currency
```

Comment:
I'm personally not a fan of ternary operators, they tend to be way harder to decipher than just if-else stataments. 
Especially when they contain non-trivial expressions in them.

Reply:
I cant really test the props @wojciech-deriv , because the current component not really using it but the onboarding component use it. More like integration test